modules/ROOT/pages/dw-binaries-functions-tobase64.adoc

```diff
@@ -18,7 +18,6 @@ Transforms a binary value into a Base64 string.
 | `content` | The binary value to transform.
 |===
 
-[[toBase64-example]]
```

Comment:
We have to add the anchor back to the docs, but I sort of doubt there's a way to add it to the dw source in a way that it gets build properly and can be single sourced in other tools, as needed. So this will prob be a manual change to the docs.